### PR TITLE
Update 'broker' to 'service broker' to be specific in the spec.

### PR DESCRIPTION
**Pull request by n3wscott**
I have updated the usage of 'broker' and 'brokers' to 'service broker' and 'service brokers' respectively. I do this only because this is suppose to be a technical spec and if there is a time where there is more than one kind of broker the language here is precise.

I also formatted the paragraphs to 80 columns where I change broker->service broker.

Please take a look! Thanks! Any feedback is welcome.

### CODE REVIEW

**Comment by n3wscott** [comment]
Not so sure about that travis error. line 1436 ends in a `|`

File: spec.md [hunk 1]
```diff
@@ -149,25 +171,25 @@ URLs:
 - `binding_id`
 - `operation`
 
-While this specification places no restriction on the set of characters used
+While this specification places no restriction on the set of characters use
```
Comment (duglin) [inline spec.md]: I think this should still be "used"

**Comment by duglin** [comment]
one minor nit, otherwise it looks good. thanks

File: spec.md (hunk 1 above)
Comment (n3wscott) [reply spec.md]: Whoops! That was a typo.

**Comment by vaikas-google** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
just one more needed

**Comment by vaikas-google** [comment]
LGTM

**Comment by avade** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by mattmcneeney** [comment]
LGTM

**Comment by vaikas-google** [comment]
LGTM

**Pull request merged**